File: Python3/2798_number_of_employees_who_met_the_target.py
# https://leetcode.cn/problems/number-of-employees-who-met-the-target/

class Solution:
    def numberOfEmployeesWhoMetTarget(self, hours, target):
        """
        Inputs:
            hours: list[int]
            target: int
        Outputs:
            res: int
        """
        res = 0
        for h in hours:
            if h >= target:
                res += 1
        return res

File: Cpp/2798_number_of_employees_who_met_the_target.cpp
// https://leetcode.cn/problems/number-of-employees-who-met-the-target/
        
class Solution {
public:
    int numberOfEmployeesWhoMetTarget(vector<int>& hours, int target) {
        int res = 0;
        for (int h: hours) {
            if (h >= target) {
                res += 1;
            }
        }
        return res;
    }
};
